apollo-client/src/main/java/com/ctrip/framework/apollo/listener/LogLevelListener.java

```diff
@@ -0,0 +1,60 @@
+package com.ctrip.framework.apollo.listener;
+
+import com.ctrip.framework.apollo.ConfigChangeListener;
+import com.ctrip.framework.apollo.model.ConfigChange;
+import com.ctrip.framework.apollo.model.ConfigChangeEvent;
+import com.ctrip.framework.apollo.spring.annotation.ApolloConfigChangeListener;
+import org.springframework.boot.logging.LogLevel;
+import org.springframework.boot.logging.LoggingSystem;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Created by lawrence on 19/12/16.
+ */
+@Service
```

Comment:
This feature relies on Spring Boot, so I think we should create a new module like `apollo-spring-boot-starter` to put Spring Boot related features.

And for each feature, we could provide a flag so that users can decide to enable it or not. We could achieve this via Spring Boot AutoConfiguration features.

Reply:
I agree. And this feature can resolve the dynamic log level change with the "LoggingSystem" help which class  in Spring Boot, so that we do not need to redefine an interface to archive this goal under spring project.